src/js/components/tasks/cards/orgActions.tsx

```diff
@@ -109,9 +109,8 @@ const OrgActions = ({
           >
             <Button
               label={i18n.t('Submit Review')}
-              variant="outline-brand"
+              variant="neutral"
```

Comment:
Hm... the tooltip does not work consistently on a `disabled` button, apparently.